README.md

```diff
@@ -44,7 +44,8 @@ Current services offered:
 ## Bounded Ports
 | PORT  | SERVICE                                       |
 |------:|:---------------------------------------------|
-| 80 | HAProxy|
+| 80    | HAProxy                                      |
+| 6379  | Redis
```

Comment:
Good spot Mr.Jonathan! This thing should be updated whenever we update the exposed ports in our server.

Unfortunately, as we discussed that we won't be doing so in the case of Redis deployment.

So let's remove this part :3